### PR TITLE
add components

### PR DESCRIPTION
**Pull request by Manasa1013**
Add components and their styles.

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Montserrat&display=swap');
+@import url("../styles.css");
+:root{
+    --grey : #808080;
```
Comment (OmkarArora) [inline alert/alert.css]: You can make this naming a little more consistent. You seem to be using two spellings "grey" and "gray", and if is using --color-name naming scheme, use it here also..

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    margin: 0.5rem;
+}
+
+.alert-success{
+    color: var(--color-green);
+    background-color: var(--background-green);
+    border-color: var(--border-green);
+    border-radius: 0.3rem;
+    width: 70%;
+    border : 1px solid transparent;
```
Comment (OmkarArora) [inline alert/alert.css]: Move padding and margin to alert class, no need to repeat them for each one.

File: badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.badge{
+    border : 1px solid transparent;
+    padding: 0.6rem 1rem;
+    background-color: var(--primary-white);
+    margin: 0.5rem;
+    /* border: 2px solid black; */
+}
+.badge__text{
```
Comment (OmkarArora) [inline badge/badge.css]: This naming scheme isn't consistent with Alert styles. There you used "-" here "__" is used. Keeping it common across the whole project.

File: grid/grid.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
+@import url("../styles.css");
+@import url("../cards/card.css");
+
+.grid--container{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+    grid-row-gap : 1rem;
+}
+.child-grid-1{
```
Comment (OmkarArora) [inline grid/grid.css]: display, width and margin being repeated for each child-grid, you can optimize this by extracting them out in a common class

**Review by OmkarArora — COMMENTED** [review]
You can remove comments after project completion. The code looks good, except some naming scheme inconsistencies in naming css variables and classes.

File: alert/alert.css (hunk 1 above)
Comment (Manasa1013) [reply alert/alert.css]: * Thanks for pointing out the mistakes in spelling. I found that color unused so removed it in the current version.

File: alert/alert.css (hunk 2 above)
Comment (Manasa1013) [reply alert/alert.css]: * Yeah, I moved 'repeated code' to one class and using it in the new version.

File: badge/badge.css (hunk 3 above)
Comment (Manasa1013) [reply badge/badge.css]: I modified most of the parts to use one naming convention, thanks a lot for pointing out that.

File: grid/grid.css (hunk 4 above)
Comment (Manasa1013) [reply grid/grid.css]: Yeah, done that. Thanks a lot, Omkar for the feedback and code review.

**Comment by sneha-amruth** [comment]
Hi Manasa, 
Your deisgn is simple and to the point. Good code organization. 
Here are some of my observations 

- You could avoid using property names as class names. 
  https://github.com/Manasa1013/ui-library/blob/6abe991b884fe869dcbd4c13711fd5c609d22c6c/index.html#L13

- Clear out the commented code which is no longer necessary
https://github.com/Manasa1013/ui-library/blob/30fe4e78581c2d002eb2cdab36830adf66b11352/navbar/nav.css#L8
https://github.com/Manasa1013/ui-library/blob/30fe4e78581c2d002eb2cdab36830adf66b11352/modal/modal.css#L4
https://github.com/Manasa1013/ui-library/blob/30fe4e78581c2d002eb2cdab36830adf66b11352/lists/list.html#L88

- Is there a reason for not including the code iframe for all the components? I think having it will make it easier for you to quickly include the component in your projects.

- Do provide some margin to the component content. Observered this in some other components as well
Screenshot 2021-04-18 at 10 48 36 PM

**Comment by Manasa1013** [comment]
Hey thanks for pointing out margin addition and commented parts, corrected it.
Maintaining relative class as such was to use it again in other apps.
I will be adding iframe code snippets in the later version, after making the code better.

**Pull request merged**